packages/map-template/src/components/Sidebar/Sidebar.jsx

```diff
@@ -21,19 +23,25 @@ const VIEWS = {
  * @param {function} props.onDirectionsOpened - Check if the directions page state is open.
  * @param {function} props.onDirectionsClosed - Check if the directions page state is closed.
  * @param {string} props.currentVenueName - The currently selected venue.
+ * @param {array} props.filteredLocationsByExternalIds - Array of locations filtered based on the external id.
+ * @param {function} props.onLocationsFilteredByExternalIds - The list of locations after filtering through the categories.
  *
 */
-function Sidebar({ currentLocation, setCurrentLocation, currentCategories, onLocationsFiltered, onDirectionsOpened, onDirectionsClosed, currentVenueName }) {
+function Sidebar({ currentLocation, setCurrentLocation, currentCategories, onLocationsFiltered, onDirectionsOpened, onDirectionsClosed, currentVenueName, filteredLocationsByExternalIds, onLocationsFilteredByExternalIds }) {
     const [activePage, setActivePage] = useState(null);
 
     const [directions, setDirections] = useState();
 
     /*
-    * React on changes on the current location.
-    */
+     * React on changes on the current location.
```

Comment:
I guess it reacts on more than current location now 😄

Reply:
✅